### PR TITLE
finish sentiment analysis functions

### PR DESCRIPTION
**Pull request by yuanxiongbear**
finish sentiment analysis function, and text cleaning function (the helper function for tweet rank function). Also change the library import.

### CODE REVIEW

**Review by huan-ds — APPROVED** [review]

**Review by JaredSplinter — COMMENTED** [review]
Hey Yuan, your code as it is takes tweets as an input and returns tweets as an output. Does this change the input dataframe? If so you may want to rewrite so that it returns a new dataframe that does not effect the original dataframe

File: pytweet/pytweet.py [hunk 1]
```diff
@@ -184,12 +192,52 @@ def sentiment_analysis(tweets):
...
+    text : np.array
+        A np.array that contains a list of strings (cleaned tweets) 
+    """
+    stopword = nltk.corpus.stopwords.words('english')
+    stopword.append('')
+    stopword.append('cont')
+    text_lc = "".join([word.lower() for word in text if word not in string.punctuation]) # remove puntuation
+    text_rc = re.sub('[0-9]+', '', text_lc)
+    tokens = re.split('\W+', text_rc)    # tokenization
+    text = [ps.stem(word) for word in tokens if word not in stopword]  # remove stopwords and stemming
```
Comment (JaredSplinter) [inline pytweet/pytweet.py:241]: ps.stem(word) is not defined, please add this before this line:

ps = SnowballStemmer('english')

**Review by JaredSplinter — CHANGES_REQUESTED** [review]
just a couple of small changes. Previous comment and line 240.

**Review by JaredSplinter — APPROVED** [review]
Thanks for making the Changes so quickly!

**Pull request merged**